batcher/aligned-batcher/src/types/batch_queue.rs

```diff
@@ -134,27 +134,25 @@ pub(crate) fn calculate_batch_size(batch_queue: &BatchQueue) -> Result<usize, Ba
 }
 
 /// This function tries to build a batch to be submitted to Aligned.
-/// Given a copy of the current batch queue, , and applyies an algorithm to find the biggest batch
+/// Given a COPY of the current batch queue, , and applyies an algorithm to find the biggest batch
```

Comment:
```
/// Given a COPY of the current batch queue, <???>, and applyies an algorithm to find the biggest batch
```

Reply:
Solved in[b019a36](https://github.com/yetanotherco/aligned_layer/pull/1512/commits/b019a3626f92b0e561fa89d64188b6f185c4f4a3).